server/src/main/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldMapper.java

```diff
@@ -114,6 +114,11 @@ public SparseVectorFieldType fieldType() {
         return (SparseVectorFieldType) super.fieldType();
     }
 
+    @Override
+    protected boolean parsesObject() {
+        return true;
```

Comment:
this mapper has been removed, effectively, looking at the javadocs above. I am thinking that we don't need to update it as we no longer support parsing this type of fields